merkletree/pad.go

```diff
@@ -63,20 +64,14 @@ func (pad *PAD) signTreeRoot(epoch uint64) {
 	pad.latestSTR = NewSTR(pad.signKey, pad.policies, m, epoch, prevHash)
 }
 
-func (pad *PAD) updateInternal(policies *Policies, epoch uint64) {
+func (pad *PAD) updateInternal(policies []byte, epoch uint64) {
 	// create STR with the policies that were actually used in the prev.
 	// Set() operation
 	pad.signTreeRoot(epoch)
 	pad.snapshots[epoch] = pad.latestSTR
 	pad.loadedEpochs = append(pad.loadedEpochs, epoch)
 	if policies != nil { // update the policies if necessary
-		vrfKeyChanged := 1 != subtle.ConstantTimeCompare(
-			pad.policies.vrfPrivateKey,
-			policies.vrfPrivateKey)
-		pad.policies = *policies
-		if vrfKeyChanged {
-			pad.reshuffle()
```

Comment:
Just saw this: https://github.com/coniks-sys/coniks-go/issues/120#issuecomment-260853410 Feel free to ignore.